### ISSUE TITLE
./wakeWordAgent File Does Not Exist

### ISSUE THREAD

**Issue by trisys3** [alexa/alexa-avs-sample-app#413]
I was able, with much difficulty, to get through most of the steps, but when I get to the wake word agent, the tutorial seems to point to a file that does not exist.

I ran the automated install (after setting all desired configuration):

    cd /path/to/AVS/tutorial/folder/
    . automated_install.sh

Then I run the NodeJS server:

    cd samples/companionService/
    npm start

I open another terminal and run the Java part:

    cd /path/to/AVS/tutorial/folder/
    cd samples/javaclient/
    mvn exec:exec

I open a 3rd terminal and try to start the wake word agent:

    cd /path/to/AVS/tutorial/folder/
    cd samples/wakeWordAgent/src/
    ./wakeWordAgent -e kitt_ai/sensory

And I get

    -bash: ./wakeWordAgent: No such file or directory

I noticed there is a `wakeWordAgent.h` and `wakeWordAgent.cpp` in that folder. I don't know much C/C++, but is it possible something wasn't built or `make`d?

**Comment by bdpiper** [comment]
I have the same problem, anyone have any ideas?

**Comment by bdpiper** [comment]
It dawned on me that I wasn't sure the microphone was plugged in when I ran the automated install program, so I re-ran the installation program. This time the wakeword program was built and works fine.

**Comment by syauqy** [comment]
I just follow this suggestion at https://github.com/alexa/alexa-avs-sample-app/issues/329 , but still got the issues of the wakewordagent install 😞

**Comment by trisys3** [comment]
I just got a microphone and speaker, so I will try all this again.

**Comment by trisys3** [comment]
I got a bluetooth microphone but have yet to obtain a bluetooth adaptor. When I do (hopefully soon), I will say whether it worked or not. In the meantime, I will close this issue.

Maybe it's because I'm oblivious, but I did not register that the wake word agent was the part that needed a microphone. It would be nice to see this in the error message itself, at the very least. This would speed up debugging. Thanks.
